### ISSUE TITLE
Fairer Minting

### ISSUE THREAD

**Issue by DragoWhooves** [ColonyPlusPlus/ColonyPlusPlus-mod#37]
Currently, 2 ingots get you 2 coins, based on the size of an ingot vs a coin that is alot of wastage

please make it more like 2 ingots get you 10 coins

**Comment by tommykent1210** [comment]
Yeah, we considered changing this, but then we'd have to re-balance the whole basegame economy :P So we decided against it!

I think we should do it as part of #38
